assets/performanceprofile/scripts/set-rps-mask.sh

```diff
@@ -33,4 +33,4 @@ function find_dev_dir {
 [ -d "${dev_dir}" ] || { sleep 5; find_dev_dir; }  # search failed, wait a little and try again
 [ -d "${dev_dir}" ] || { echo "${dev_dir}" directory not found >&2 ; exit 0; } # the interface disappeared, not an error
 
-echo "${mask}" > "${dev_dir}"/queues/rx-0/rps_cpus
+find "${dev_dir}"/queues -type f -name rps_cpus -exec sh -c "echo ${mask} | cat > {}" \;
```

Comment:
I just wonder if a plain bash `for` loop would be better than calling `find + cat`. Did we try and benchmark the for loop approach?

Reply:
There is no significant difference between using `for` and `find+cat`:
```
sh-4.4# time for i in /sys/class/net/*/queues/rx-*/rps_cpus; do sh -c echo test > /dev/null; done

real    0m0.292s
user    0m0.080s
sys     0m0.209s
sh-4.4# time find /sys/class/net/*/queues/ -type f -name rps_cpus -exec sh -c "echo test > /dev/null" \;

real    0m0.292s
user    0m0.104s
sys     0m0.177s
```

But!
There is a huge difference when we're not forking into a new shell process:
```
sh-4.4# time for i in /sys/class/net/*/queues/rx-*/rps_cpus; do echo test > /dev/null; done

real    0m0.004s
user    0m0.001s
sys     0m0.003s
```
Notice we remove the `sh -c`

Reply:
yes, this is why I mentioned this approach. I assumed to do a for loop fully within spawning processes (so no `sh -c`). I should have been explicit since the beginning, sorry about that.

Reply:
I believe the pure-shell for loop has its merit if we have reasonnable guarantees the glob expression will be valid for a long time. E.g. if the path is ALWAYS like
```bash
/sys/class/net/FOO/queues/rx-NUM/rps_cpus
/sys/class/net/BAR/queues/rx-NUM/rps_cpus
/sys/class/net/BAZ/queues/rx-NUM/rps_cpus
```

and like never ever nested devices:
```
/sys/class/net/FOO/BAR/queues/rx-NUM/rps_cpus
```

Reply:
Yes. This is a stable API which I don't think would change

Reply:
cool, then in a followup PR we can perhaps optimize further!

Reply:
Why followup PR? I already apply the change

Reply:
ah, great!